### ISSUE TITLE
Links are not focusable and thus are inaccessible to keyboard/screen reader users

### ISSUE THREAD

**Issue by cassidyjames** [ubuntu/app-center#1591]
If I'm on an app's detail page, I cannot use the keyboard to tab to the listed links like the developer's website or contact link. This makes them inaccessible without a mouse, and invisible to screen readers.
